src/OpenTelemetry.Extensions.PersistentStorage/README.md

```diff
@@ -1,12 +1,12 @@
-# Persistent Storage Interface and Implementation
+# Persistent Storage file based implementation
 
 [![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Extensions.PersistentStorage.svg)](https://www.nuget.org/packages/OpenTelemetry.Extensions.PersistentStorage)
 [![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Extensions.PersistentStorage.svg)](https://www.nuget.org/packages/OpenTelemetry.Extensions.PersistentStorage)
 
-This package provides both the interface and implementation of persistent
-storage. It is an experimental component which can be used by OpenTelemetry
-Exporters to provide reliable data delivery. Eventually this component should
-get splitted into the abstract interface and concrete implementations.
+This package provides file based implementation of
```

Comment:
Minor, consider "This package provides an implementation of ... based on local file system."